build.gradle

```diff
@@ -7,14 +7,17 @@ buildscript {
         maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:2.3.1'
         classpath 'io.fabric.tools:gradle:1.+'
     }
 }
 
 allprojects {
     repositories {
         jcenter()
+        flatDir {
+            dirs 'libs'
+        }
```

Comment:
This ensures that we can either fetch from gradle or put directly in here the `aar` thus skipping fetching from gradle.